src/oslo_aero_ocean.F90

```diff
@@ -246,31 +244,18 @@ subroutine oslo_aero_dms_emis(ncol, lchnk, u, v, zm, ocnfrc, icefrc, sst, fdms,
        open_ocn(:ncol) = ocnfrc(:ncol) * (1._r8-icefrc(:ncol))
 
        !start with midpoint wind speed
-       u10m(:ncol)=sqrt(u(:ncol)**2 + v(:ncol)**2)
-
-       if (method_oslo) then
-          ! move the winds to 10m high from the midpoint of the gridbox:
-          u10m (:ncol) = u10m(:ncol)*log(10._r8/z0)/log(zm(:ncol)/z0)
-          rk600(:ncol) = (0.222_r8*(u10m(:ncol)*u10m(:ncol))) + (0.333_r8*u10m(:ncol))        ! [cm/hr]
-          flux (:ncol) = 2.778e-15*cnst_mw(pndx_fdms)*rk600(:ncol)*open_ocn(:ncol)*odms(:ncol) ! [kg m-2 s-1]
-       else if (method_hamocc) then
-          t(:ncol)     = sst(:ncol)-273.15_r8
-          u10m (:ncol) = u10m(:ncol)*log(10._r8/z0)/log(zm(:ncol)/z0)
-          scdms(:ncol) = 2855.7+  (-177.63 + (6.0438 + (-0.11645 + 0.00094743*t(:ncol))*t(:ncol))*t(:ncol))*t(:ncol)
-          kwdms(:ncol) = open_ocn(:ncol) * Xconvxa *u10m(:ncol)**2*(660./scdms(:ncol))**0.5
-          flux (:ncol) = 62.13*kwdms(:ncol)*1e-9*odms(:ncol)
-       endif
-
-       cflx(:ncol,pndx_fdms) = flux(:ncol)
-
+       u10m(:ncol) = sqrt(u(:ncol)**2 + v(:ncol)**2)
+
+       ! method_hamocc is the only one supported
+       t(:ncol)     = sst(:ncol)-273.15_r8
+       u10m (:ncol) = u10m(:ncol)*log(10._r8/z0)/log(zm(:ncol)/z0)
+       scdms(:ncol) = 2855.7+  (-177.63 + (6.0438 + (-0.11645 + 0.00094743*t(:ncol))*t(:ncol))*t(:ncol))*t(:ncol)
+       kwdms(:ncol) = open_ocn(:ncol) * Xconvxa *u10m(:ncol)**2*(660./scdms(:ncol))**0.5
+       flux (:ncol) = 62.13*kwdms(:ncol)*1e-9*odms(:ncol)
```

Comment:
These naked constants should be defined as parameters (or used from physconst). They should also be defined as `r8` numbers.